DSharpPlus.HttpInteractions.AspNetCore/EndpointRouteBuilderExtensions.cs

```diff
@@ -0,0 +1,79 @@
+using System.Buffers;
+using System.Diagnostics.CodeAnalysis;
+using System.Net.Mime;
+using System.Text;
+using DSharpPlus.Net.HttpInteractions;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.Primitives;
+using Microsoft.Net.Http.Headers;
+
+namespace DSharpPlus.HttpInteraction.AspNetCore;
+
+public static class EndpointRouteBuilderExtensions
+{
+    /// <summary>
+    /// Adds handling of Discords http interactions to the specified route.
```

Comment:
Potentially reword closer to the following? 

"Registers an endpoint to handle HTTP-based interactions from Discord"
Reads a bit more naturally to me, but I'm open to feedback on this. (Also, "Discords" is missing an apostrophe)